tests/test_model.py

```diff
@@ -1179,27 +1179,31 @@ def test_latest_components_queryset(client, api_path, stored_proc):
         # A NEVRA like "PyYAML version 1.2.3 with no release or architecture"
         # could be a PyPI package, or an upstream RPM
         # We want to see the latest version of both components
-        # Create PyPI components for src architecture only
+        # Create PyPI components once per loop, like src architecture components
         # with the same namespace, name and version as UPSTREAM noarch components
         # and no release, arch, or software_build
+        # TODO: We should use RPMMOD instead of PyPI here, but then the test fails
+        #  because we reintroduced a bug we fixed previously
+        #  Whe hide results if two components have the same name and namespace
+        #  but different types or arches, for example a source RPM and its related RPM module
```

Comment:
These comments need fixing.

```suggestion
        # Create RPMMOD components once per loop, like src architecture components
        # with the same namespace, name and version as UPSTREAM noarch components
        # and no release, arch, or software_build. Otherwise we hide results if two 
        # components have the same name and namespace but different types or arches, 
        # for example a source RPM and its related RPM module
```